### PR TITLE
Docs - Remove Deploy to GH pages action and serve docs using iota-wiki-cli

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Migrated to using iota-wiki-cli to have the same local setup as on the wiki
* Removed deploy-docs-to-gh-pages action as we're moving to wiki.iota.org
* Removed Docusaurus files which are no longer used

Replaces https://github.com/iotaledger/wasp/pull/423

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix

## How the change has been tested

Docs site was mounted locally within the wiki using iota-wiki-cli.

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -1,32 +1,46 @@
-# Website
+# Documentation
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+The documentation is built using [Docusaurus 2](https://docusaurus.io/). The deployment is done through a centralized build from [IOTA WIKI](https://github.com/iota-community/iota-wiki). To run a local instance the [IOTA WIKI CLI](https://github.com/iota-community/iota-wiki-cli) is used.
+
+## Prerequisites
+
+- [Node.js v14.14+](https://nodejs.org/en/)
+- [yarn](https://yarnpkg.com/getting-started/install)
```
Comment (jorgemmsilva) [inline documentation/README.md]: I guess yarn is not a pre-requisite :p

File: documentation/docusaurus.config.js [hunk 2]
```diff
@@ -1,71 +0,0 @@
-const lightCodeTheme = require('prism-react-renderer/themes/github');
```
Comment (jorgemmsilva) [inline documentation/docusaurus.config.js:1]: Is this file supposed to be removed?

**Review by jorgemmsilva — APPROVED** [review]
please confirm that removing `docusaurus.config.js` is intended.
otherwise LGTM

File: documentation/docusaurus.config.js (hunk 2 above)
Comment (lucas-tortora) [reply documentation/docusaurus.config.js:1]: Hi @jorgemmsilva , yes, it is .  The iota-wiki-cli uses the wiki's config which it combines with documentation/EXTERNAL_DOCS_CONFIG and documentation/EXTERNAL_DOCS_DROPDOWN_CONFIG .

File: documentation/README.md (hunk 1 above)
Comment (lucas-tortora) [reply documentation/README.md]: I guess not, it should be npm in this case.  

```suggestion
- [npm](https://npmjs.com)
```

**Pull request merged**